src/frontend/src/pages/CreateWorkPackagePage/CreateWorkPackageForm.tsx

```diff
@@ -29,11 +31,11 @@ const CreateWorkPackageForm: React.FC = () => {
   const history = useHistory();
   const auth = useAuth();
   const toast = useToast();
+  const query = useQuery();
 
   const { isLoading, mutateAsync } = useCreateSingleWorkPackage();
-
+  const [wbsNum, setWbsNum] = useState(query.get('wbsNum') || '');
```

Comment:
good, but now you have to use this wbsNum in the actual payload instead of the outdated wbsNum in the useForm(erase all instances of that from this file because the useState is replacing it.